kuma/wiki/jinja2/wiki/document.html

```diff
@@ -243,6 +243,16 @@
             <!-- center: main article content -->
             <div id="wiki-content" class="{{ content_class }} wiki-column text-content">
 
+              {% if settings.MAINTENANCE_MODE and render_raw_fallback %}
+                <div id="doc-render-raw-fallback" class="warning">
+                {% trans url='/maintenance-mode/' %}
+                  This document has not yet been rendered, and cannot be
+                  rendered while MDN is in <a href="{{ url }}">maintenance
+                  mode</a>. Please check back when maintenance has concluded.
+                {% endtrans %}
+                </div>
+              {% endif %}
```

Comment:
👍 I think this is the right solution, even if it looks really bad when you use the sample DB, which currently has no rendered docs ☹️